src/main/java/uk/gov/companieshouse/accounts/association/utils/MapperUtil.java

```diff
@@ -54,22 +54,6 @@ public Association enrichAssociationWithUserDetails(final Association associatio
         return association;
     }
 
-    public Association enrichInvitations(final Association association) {
```

Comment:
you would still need this for invitations endpoint.

Reply:
Good point - added it back